lib/generators/solidus_starter_frontend/solidus_starter_frontend_generator.rb

```diff
@@ -2,27 +2,49 @@
 
 require 'bundler'
 
+# `rspec:install` requires these libraries to run.
+require 'stringio'
+require 'tmpdir'
```

Comment:
As mentioned by the comment, these libraries are required to invoke `rspec:install`. I haven't figure out why this has to be done, but we might eventually remove these requires once we convert the gem into a rails template.